src/utils/deprecated/deprecated.tsx

```diff
@@ -5,14 +5,34 @@
 
 import React, { useEffect } from 'react';
 
+interface IDeprecatedComponentWarning {
+  NewComponent: React.ComponentType<any>;
```

Comment:
Is this intentionally capitolized?

Reply:
Yes, it is intentionally capitalized because it's a component. In the new commit, I decided not to pass the component, just the component name, to be more flexible